### PR TITLE
Update coverage

### PR DESCRIPTION
**Pull request by victoreronmosele**
## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by jeremiahlukus** [comment]
Need to run 

flutter pub run import_sorter:main

Then flutter analyze

**Comment by victoreronmosele** [comment]
> Need to run
> 
> flutter pub run import_sorter:main
> 
> Then flutter analyze

Thanks, I've done that.

**Comment by jeremiahlukus** [comment]
Now 
flutter format -l  120 

Sorry it’s not the normal formatting I feel 80 is too small .

**Comment by victoreronmosele** [comment]
Oh, okay. On it.

File: test/auth/infrastructure/oauth2_interceptor_test.dart [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+          HttpStatus.httpVersionNotSupported,
+          HttpStatus.variantAlsoNegotiates,
+          HttpStatus.insufficientStorage,
+          HttpStatus.loopDetected,
+          HttpStatus.notExtended,
+          HttpStatus.networkAuthenticationRequired,
+          HttpStatus.networkConnectTimeoutError,
+        ];
+
+        for (final non401ErrorCode in httpErrorCodesThatAreNot401) {
```
Comment (jeremiahlukus) [inline test/auth/infrastructure/oauth2_interceptor_test.dart:177]: Nice way to approach this

**Pull request merged**